app/components/Header.tsx

```diff
@@ -4,13 +4,16 @@ import {
   HOW_TO_LIST_PROJECT_URL,
   RESOURCES_LINK,
   DISCORD_LINK,
+  SIGNIN_LINK,
+  SIGNUP_LINK,
+  LINK_FORMATS,
 } from "../utils/constants";
 import CallToAction from "./CallToAction";
 import NavLink from "./NavLink";
-import logo from "~/images/logo_white.png";
+import logo from "~/images/logo.svg";
```

Comment:
**todo:** please revert this change, we want to display the new "Open-Source Hub" logo instead of "OSS Port". I apologize that the Figma file doesn't have the latest logo yet, I'll get that fixed as soon as possible.

Reply:
Logo was updated as a new one.